src/main/java/de/tum/in/www1/artemis/web/rest/QuizSubmissionResource.java

```diff
@@ -155,9 +155,10 @@ public ResponseEntity<Result> submitForPractice(@PathVariable Long exerciseId, @
         }
 
         quizExercise.setQuizPointStatistic(null);
-        quizExercise.setCourse(null);
 
         messagingService.broadcastNewResult(result.getParticipation(), result);
+
+        quizExercise.setCourse(null);
```

Comment:
Why is this change necessary?

Reply:
Otherwise we cannot determine if the feedbacks have to be filtered before broadcasting